crates/requirements-txt/src/lib.rs

```diff
@@ -801,9 +799,7 @@ fn parse_requirement_and_hashes(
         })?;
 
     let hashes = if has_hashes {
-        let hashes = parse_hashes(content, s)?;
-        eat_trailing_line(content, s)?;
```

Comment:
Not necessary, I think, because the next step after parsing an entry is to eat the trailing line.